src/main/java/entities/Hotel.java

```diff
@@ -0,0 +1,16 @@
+package entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.ArrayList;
+
+@Data
+@AllArgsConstructor
+public class Hotel {
```

Comment:
Create class AbstractObject - parent for all entity classes. It will contain `id` field